src/app/components/address-displayer/address-displayer.tsx

```diff
@@ -19,7 +21,11 @@ export function AddressDisplayer({ address }: AddressDisplayerProps) {
   return (
     <>
       {groupByFour(address).map((letterGroup, index) => (
-        <AddressDisplayerLayout key={index} isEven={isEven(index + 1)}>
+        <AddressDisplayerLayout
+          key={index}
+          isEven={isEven(index + 1)}
+          data-testid={CommonSelectors.AddressDisplayerPart}
```

Comment:
There is already an id `HomePageSelector.AddressDisplayer`, why do we need one here too?

Reply:
moved this to shared-components.selectors